### PR TITLE
Clarify URL validation error on invalid http url

### PR DESCRIPTION
**Pull request by jjbuchan**
Fixes https://github.com/racker/ele/issues/3015

### CODE REVIEW

File: lib/valve.js [hunk 1]
```diff
@@ -804,9 +804,12 @@ Chain.prototype.isHttpUrl = function() {
   this._pushValidator({
     name: 'isUrl',
     func: function(value, baton, callback) {
-      if (!check.isURL(value, { protocols: ['http', 'https'], require_protocol: true })) {
+      if (!check.isURL(value)) {
         callback('Invalid URL');
         return;
+      } else if (!check.isURL(value, { protocols: ['http', 'https'], require_protocol: true })) {
+        callback('Invalid URL.  URL must begin with http or https protocol.');
```
Comment (jirwin) [inline lib/valve.js]: I'm not sure about this error message. We are actually requiring that urls start with `http://` or `https://`, not just the protocol. Maybe `Invalid URL. URL must begin with http:// or https://.`

**Comment by jirwin** [comment]
Just a small, non-blocking nit on the error message. Other than that, +1

**Comment by jjbuchan** [comment]
@jirwin updated.

**Comment by jirwin** [comment]
+1

**Pull request merged**